index.js

```diff
@@ -103,4 +104,9 @@ app.on('before-quit', () => {
 	if (!mainWindow.isFullScreen()) {
 		storage.set('lastWindowState', mainWindow.getBounds());
 	}
+
+	// save the system's theme to determine if it changed on next startup.
+	if (process.platform === 'darwin') {
+		storage.set('lastSystemTheme', osxAppearance.isDarkMode());
```

Comment:
I'm with @paulmolluzzo on this. The system theme doesn't matter. If I set the Anatine theme to dark, I want it to be dark on the next startup. Independent of whether I changed the system theme in the mean time.